specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-06-01/examples/AgentPools_Update.json

```diff
@@ -0,0 +1,72 @@
+{
+  "parameters": {
+    "api-version": "2019-06-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "agentPoolName": "agentpool1",
+    "parameters": {
+      "properties": {
+        "orchestratorVersion": "",
+        "enableAutoScaling": true,
+        "minCount": 2,
+        "maxCount": 2,
+        "vmSize": "Standard_DS1_v2",
+        "osType": "Linux",
+        "nodeTaints": [
+          "Key1=Value1:NoSchedule"
+        ],
+        "scaleSetPriority": "Low",
+        "scaleSetEvictionPolicy": "Delete"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/managedClusters/clustername1/agentPools/agentpool1",
+        "type": "Microsoft.ContainerService/managedClusters/agentPools",
+        "name": "agentpool1",
+        "properties": {
+          "provisioningState": "Succeeded",
```

Comment:
If this is an example of updating agent pool, the provisioningState in the response could be "Updating", "Upgrading". It cannot be "Succeed"

Reply:
this is copied from the Create/Update response - so I'd suggest if that doesn't match the API definition it'd be worth the service team opening a separate PR for that, since it sounds like the API definition could be out of date?